packages/cli/src/commands/generate.js

```diff
@@ -76,7 +76,7 @@ export default {
 
     // Forcing static target anyway
     config.target = TARGETS.static
-    consola.warn(`For using \`nuxt generate\`, your have to set \`target: 'static'\` in your \`nuxt.config\`\n       👉 Learn more about it on https://go.nuxtjs.dev/static-target`)
+    consola.warn(`When using \`nuxt generate\`, you have to set \`target: 'static'\` in your \`nuxt.config\`\n       👉 Learn more about it on https://go.nuxtjs.dev/static-target`)
```

Comment:
```suggestion
    consola.warn(`When using \`nuxt generate\`, you should set \`target: 'static'\` in your \`nuxt.config\`\n       👉 Learn more about it on https://go.nuxtjs.dev/static-target`)
```

**suggestion**: this is a warning before forcing - so you don't *have* to set target